package.json

```diff
@@ -53,9 +59,16 @@
     "dynamodb-local": "0.0.31"
   },
   "devDependencies": {
+    "@babel/cli": "7.18.9",
+    "@babel/core": "7.18.9",
     "@shelf/babel-config": "1.0.2",
     "@shelf/eslint-config": "2.18.0",
     "@shelf/prettier-config": "1.0.0",
+    "@shelf/tsconfig": "0.0.8",
+    "@types/aws-sdk": "2.7.0",
+    "@types/cwd": "^0.10.0",
+    "@types/jest": "28.1.6",
+    "@types/node": "16",
     "eslint": "8.20.0",
     "husky": "8.0.1",
     "jest": "28.1.3",
```

Comment:
There is a difference in versions between the `jest` and its types package, it would be better to have the same version.